test/saithrift/src/switch_sai_rpc_server.cpp

```diff
@@ -3242,11 +3242,11 @@ class switch_sai_rpcHandler : virtual public switch_sai_rpcIf {
       sai_u32_list_t *lane_list_num;
 
       port_hw_lane.id = SAI_PORT_ATTR_HW_LANE_LIST;
-      port_hw_lane.value.u32list.list = (uint32_t *) malloc(sizeof(uint32_t) * 4);
-      port_hw_lane.value.u32list.count = 4;
+      port_hw_lane.value.u32list.list = (uint32_t *) malloc(sizeof(uint32_t) * 8);
+      port_hw_lane.value.u32list.count = 8;
       port_api->get_port_attribute(port_id, 1, &port_hw_lane);
 
-      thrift_attr_list.attr_count = 4;
```

Comment:
i think this line is not needed

Reply:
You're correct.
All the assign statements `thrift_attr_list.attr_count = xx` are not necessary.
Will update after verified it.